ZuluContent/SkillHandlers/Stealth.cs

```diff
@@ -4,87 +4,89 @@
-				/* Barbed	*/	{ 0,	0,		0,		0,		0,		0,		0 },
-				/* Ring		*/	{ 0,	5,		0,		10,		15,		25,		0 },
-				/* Chain	*/	{ 0,	0,		10,		0,		15,		25,		0 },
-				/* Plate	*/	{ 5,	5,		10,		10,		15,		25,		0 },
-				/* Dragon	*/	{ 0,	5,		10,		10,		15,		25,		0 }
-			};
+        public static int[,] ArmorTable
+        {
+            get { return m_ArmorTable; }
+        }
 
-		public static int GetArmorRating( Mobile m )
-		{
-			return (int)m.ArmorRating;
-		}
+        private static int[,] m_ArmorTable = new[,]
+        {
+            //	Gorget	Gloves	Helmet	Arms	Legs	Chest	Shield
+            /* Cloth	*/ {0, 0, 0, 0, 0, 0, 0},
+            /* Leather	*/ {0, 0, 0, 0, 0, 0, 0},
+            /* Studded	*/ {2, 2, 0, 4, 6, 10, 0},
+            /* Bone		*/ {0, 5, 10, 10, 15, 25, 0},
+            /* Spined	*/ {0, 0, 0, 0, 0, 0, 0},
+            /* Horned	*/ {0, 0, 0, 0, 0, 0, 0},
+            /* Barbed	*/ {0, 0, 0, 0, 0, 0, 0},
+            /* Ring		*/ {0, 5, 0, 10, 15, 25, 0},
+            /* Chain	*/ {0, 0, 10, 0, 15, 25, 0},
+            /* Plate	*/ {5, 5, 10, 10, 15, 25, 0},
+            /* Dragon	*/ {0, 5, 10, 10, 15, 25, 0}
+        };
 
-		public static TimeSpan OnUse( Mobile m )
-		{
-			if ( !m.Hidden )
-			{
-				m.SendLocalizedMessage( 502725 ); // You must hide first
-			}
-			else if ( m.Skills[SkillName.Hiding].Base < HidingRequirement )
-			{
-				m.SendLocalizedMessage( 502726 ); // You are not hidden well enough.  Become better at hiding.
-				m.RevealingAction();
-			}
-			else if( !m.CanBeginAction( typeof( Stealth ) ) )
-			{
-				m.SendLocalizedMessage( 1063086 ); // You cannot use this skill right now.
-				m.RevealingAction();
-			}
-			else
-			{
-				int armorRating = GetArmorRating( m );
+        public static int GetArmorRating(Mobile m)
+        {
+            return (int) m.ArmorRating;
+        }
 
-				if( armorRating >= 26 ) //I have a hunch '42' was chosen cause someone's a fan of DNA
-				{
-					m.SendLocalizedMessage( 502727 ); // You could not hope to move quietly wearing this much armor.
-					m.RevealingAction();
-				}
-				else if( m.CheckSkill( SkillName.Stealth, -20.0 + armorRating * 2, 80.0 + armorRating * 2 ) )
-				{
-					int steps = (int)(m.Skills[SkillName.Stealth].Value / 10.0);
+        public static TimeSpan OnUse(Mobile m)
+        {
+            if (!m.Hidden)
+            {
+                m.SendLocalizedMessage(502725); // You must hide first
+            }
+            else if (m.Skills[SkillName.Hiding].Base < HidingRequirement)
+            {
+                m.SendLocalizedMessage(502726); // You are not hidden well enough.  Become better at hiding.
+                m.RevealingAction();
+            }
+            else if (!m.CanBeginAction(typeof(Stealth)))
+            {
+                m.SendLocalizedMessage(1063086); // You cannot use this skill right now.
+                m.RevealingAction();
+            }
+            else
+            {
+                int armorRating = GetArmorRating(m);
```

Comment:
Looks like this didn't exist in ZH? I think this would really limit thieves ability to hunt/pvp

Reply:
Yeah this is just white-space chances from auto-formatting, I haven't actually changed anything in this file